### PR TITLE
Amls 5529 - AuthAction like FE and few extras

### PR DESCRIPTION
**Pull request by slavvco**
In this PR added AuthAction like in Frontend, AMLSConnector, AMLSFESessionRepository, release note generator and PR template.

!!! IMPORTANT !!! 
EAB micro-service is not connecting to AMLS FE repository, therefore it is not using AuthAction for authorisation but it's using session based solution. Auth implementation needs to be updated in the future to the similar one like in AMP, UserAnswers need to be updated too, removed SessionBased repository references and AMLSFERepository switched on instead. Also binding needs to be updated in configuration.

### CODE REVIEW

File: app/connectors/AMLSConnector.scala [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import uk.gov.hmrc.play.bootstrap.http.HttpClient
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class AMLSConnector @Inject()(config: Configuration,
+                              implicit val httpClient: HttpClient)
+                             (implicit ec: ExecutionContext) {
+
+  private val baseUrl                 = config.get[Service]("microservice.services.amls-frontend")
+  private[connectors] val url: String = s"$baseUrl/amp"
```
Comment (ajchilts) [inline app/connectors/AMLSConnector.scala]: 'amp' should be replaced with 'eab'.

File: app/controllers/actions/IdentifierAction.scala [hunk 2]
```diff
@@ -20,46 +20,82 @@ import com.google.inject.Inject
...
     }
   }
 }
 
-class SessionIdentifierAction @Inject()(
-                                         config: FrontendAppConfig,
-                                         val parser: BodyParsers.Default
-                                       )
-                                       (implicit val executionContext: ExecutionContext) extends IdentifierAction {
+class SessionIdentifierAction @Inject()(config: FrontendAppConfig,
```
Comment (ajchilts) [inline app/controllers/actions/IdentifierAction.scala]: Maybe add a TODO to remove this when we hook up to FE?

File: app/connectors/AMLSConnector.scala (hunk 1 above)
Comment (slavvco) [reply app/connectors/AMLSConnector.scala]: done

File: app/controllers/actions/IdentifierAction.scala (hunk 2 above)
Comment (slavvco) [reply app/controllers/actions/IdentifierAction.scala]: done

**Pull request merged**